treatment-service/config/config_test.go

```diff
@@ -19,7 +19,7 @@ func TestDefaultConfigs(t *testing.T) {
 			AllowedOrigins:   []string{"*"},
 			OpenAPISpecsPath: ".",
 		},
-		ProjectIds: nil,
+		ProjectIds: []string{},
```

Comment:
This change is made because the default config object created by viper now specifies an empty array instead of nil by default when that field is unspecified.